lisa/tools/nfs.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import PurePosixPath
+from typing import List
+
+from assertpy.assertpy import assert_that
+
+from lisa.executable import Tool
+from lisa.operating_system import SLES, Debian, Redhat
+from lisa.tools import Mount
+from lisa.util import LisaException
+
+
+class NFS(Tool):
+    @property
+    def command(self) -> str:
+        return ""
+
+    def can_install(self) -> bool:
+        return True
+
+    def setup_server(self, client_ips: List[str], dir_name: str) -> None:
+        # run installation
+        self._install()
+
+        # create directory to share
+        self.node.execute(f"chown -R nobody:nogroup {dir_name}", sudo=True)
+
+        # add client ip to /etc/exports file
+        for client_ip in client_ips:
+            self.node.execute(
+                f"echo '{dir_name} {client_ip}(rw,sync,no_subtree_check)' "
+                ">> /etc/exports",
+                shell=True,
+                sudo=True,
+            )
+
+        # restart nfs service
+        if isinstance(self.node.os, Redhat):
+            self._disable_firewall()
```

Comment:
can use from microsoft.testsuites.network.common import stop_firewall

Reply:
Don't import test suite from a tool. But we can create a virtual tool called firewall.